examples/01_plotting/plot_demo_glass_brain_extensive.py

```diff
@@ -30,45 +30,41 @@
 print('Datasets shipped with nilearn are stored in: %r' % datasets.get_data_dirs())
 
 ###############################################################################
-# Let us now retrieve a motor task contrast maps corresponding to second subject
-# from a localizer experiment
-tmap_filenames = datasets.fetch_localizer_button_task()['tmaps']
-print(tmap_filenames)
-
-###############################################################################
-# tmap_filenames is returned as a list. We need to take first one
-tmap_filename = tmap_filenames[0]
+# Let us now retrieve a motor task contrast map
+# corresponding to a group one-sample t-test
 
+motor_images = datasets.fetch_neurovault_motor_task()
```

Comment:
We should probably print the value of motor_images here, as it used to be done. It's didactic for users.